kuma/wiki/forms.py

```diff
@@ -327,63 +356,57 @@ def clean_current_rev(self):
             # If there's no document yet, just bail.
             return current_rev
 
-    def save_section(self, creator, document, **kwargs):
```

Comment:
Looks like we're removing the section-editing functionality, but we've still left in some comments, logical paths (`if (self.section_id ...`) and tests for it?